src/backend/d3d12/CommandBufferD3D12.cpp

```diff
@@ -52,12 +53,17 @@ namespace d3d12 {
             DescriptorHeapHandle cbvSrvUavGPUDescriptorHeap;
             DescriptorHeapHandle samplerGPUDescriptorHeap;
             std::array<BindGroup*, kMaxBindGroups> bindGroups = {};
+            bool inCompute;
```

Comment:
`bool inCompute = false`;

or perhaps `bindingTracker.SetInComputePass(false);` in `Command::BeginRenderPass`